tidy3d/plugins/dispersion/fit_fast.py

```diff
@@ -721,7 +675,7 @@ def fit(
             Best fitting result: (dispersive medium, weighted RMS error).
         """
 
-        omega = 2 * np.pi * HBAR * self.freqs[::-1]
+        omega = PoleResidue.angular_freq_to_eV(PoleResidue.Hz_to_angular_freq(self.freqs[::-1]))
```

Comment:
One interesting thing -- this line appears to be responsible for the different results now for the following materials:


Ag-RakicLorentzDrude1998
AlAs-FernOnton1971
Al-Rakic1995
GaAs-Skauli2003
InP-Pettit1965
Si3N4-Luke2015PMLStable
Si3N4-Luke2015Sellmeier
Y2O3-Nigara1968


Because, floating-point multiplication is non-commutative. Previously, I multiplied by HBAR first, then 2 pi. The difference in results seems minor though, just a shortcoming of the approach that it is sensitive to such things.

Reply:
not quite, actually. I guess the previous version was multiplying 2*pi*HBAR first, then multiplying by the frequency. The new version is probably more sensible, since HBAR*frequency is of order 1

Reply:
yeah, I think the extrema finding will only make a difference when you bound the loss range by a finite nonzero value.

Reply:
so do you think i should leave this part as it is, or revert to the previous `2 * np.pi * HBAR * `?

Reply:
so, I think that the new PR improves some of these fits by fixing this issue

Reply:
I think you should leave it as is